torch/_inductor/compile_fx.py

```diff
@@ -1543,6 +1547,9 @@ def compile_fx(
     NB: This function TAKES OWNERSHIP of the input ``model_`` and can potentially
     mutate it!  Make a copy if you need to preserve the original GraphModule.
     """
+    
+    # [See NOTE] Unwrapping subclasses AOT
+    unwrap_tensor_subclass_parameters(model_)
```

Comment:
huh... this will run whenever `compile_fx()` is called, which is basically the top-level function invoked in inductor when we run `torch.compile`. Why do we want it here?

Reply:
I was imaging more that once you have `unwrap_tensor_subclass_parameters` getting implicitly used by `export.run_decompositions()`, then in theory full AOTI support should come for free - since the function that we are passing to inductor in the AOTI case has already been parametrized. So we just need some testing that the E2E case works properly.

Since torchAO are some of the main customers, we probably want to make sure their export cases work automatically with AOTI too

Reply:
Yup! I talked with @desertfire  bit about this, so the aot_export_module still exist in aot_inductor because originally aoti code was designed to handle any fx graph (including export) hence this fork exist. But we both agreed that in the long term, run-decomp should consume this part.